### PR TITLE
Add license check doc

### PR DESCRIPTION
**Pull request by tiagojsag**

### CODE REVIEW

File: source/developers-guide/index.html [hunk 1]
```diff
@@ -16,6 +16,7 @@
     <li><a href="{{ site.url }}/developers-guide/shopware-5-cli-commands/">CLI commands</a></li>
     <li><a href="{{ site.url }}/developers-guide/controller/">Shopware controllers</a></li>
     <li><a href="{{ site.url }}/developers-guide/services/">Plugin services</a></li>
+    <li><a href="{{ site.url }}/developers-guide/shopware-5-plugin-license/">License check in plugins</a></li>
```
Comment (bcremer) [inline source/developers-guide/index.html]: Please remove the shopware-5 from the file path.

File: source/developers-guide/shopware-5-plugin-license/index.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+## Overview
+
+Should you wish to, you can have your plugin run a license check when used in a Shopware shop. This will ensure that the target shop has permission to use your plugin. This document covers the steps necessary to implement this check. 
+
+## Requesting the license check
+
+To request a license check, you must first login into your [Shopware account](http://account.shopware.com) and request a license validation in the detail page of your plugin. Once you have done so, you will be given a code snippet similar to this: 
+
+```
```
Comment (bcremer) [inline source/developers-guide/shopware-5-plugin-license/index.md]: Remove the indentation. Code fences are enough. See comments in the original PR.

**Comment by bcremer** [comment]
Please get rid of the unnecessary shopware in the URLs `/developers-guide/shopware-plugin-license/` -> `/developers-guide/plugin-license/`.

**Comment by tiagojsag** [comment]
@bcremer fixed the shopware thingy and squashed the commits

**Pull request merged**